src/view/concept-card.inc.twig

```diff
@@ -36,6 +36,9 @@
         {%- endif -%}
         </div>
         <div class="col-sm-8" id="concept-label">
+        {% if concept.hasXlLabel %}
+          {{ include('xl-label.inc.twig', {xlLabel: concept.xlLabel}) }}
+        {% endif %}
         {% if concept.notation %}
           <h1 id="concept-preflabel"
```

Comment:
The prefLabel could have `aria-describedby="id-of-the-tooltip-html"` attribute to ling the label h1-element to the tooltip info. It is possibly easier to wrap the xl-label.inc.twig template inside a `<div id="id-of-the-tooltip-html">` element to keep the assigning of unique identifiers in the control of the concept-card.inc.twig